content/backmatter.tex

```diff
@@ -18,56 +18,69 @@ \chapter{Writing OpenSHMEM Programs}
                 {example_code/hello-openshmem-c.output}
 \vspace{0.1in}
 \end{minipage}
 
-\openshmem also has a \Fortran API, therefore listing~\ref{openshmem-hello-f90} provides the same program written in \Fortran:
+\clearpage %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\begin{deprecate}
+\openshmem also provides a \Fortran API. Listing~\ref{openshmem-hello-f90} shows a similar program written in \Fortran.
 
 \begin{minipage}{\linewidth}
 \vspace{0.1in}
-\numberedlisting{caption={``Hello World'' example program (Fortran)},label=openshmem-hello-f90,language=OSH2+F}
+\numberedlisting{caption={``Hello World'' example program in \Fortran},label=openshmem-hello-f90,language=OSH2+F}
                 {example_code/hello-openshmem.f90}
-\outputlisting{language=bash,caption={Expected output from the program in Listing~\ref{openshmem-hello-f90} (4 processors)}}
+\outputlisting{language=bash,caption={Possible ordering of expected output with 4 \acp{PE} from the program in Listing~\ref{openshmem-hello-f90}}}
                 {example_code/hello-openshmem-f90.output}
 \vspace{0.1in}
 \end{minipage}
+\end{deprecate}
+
+\clearpage %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-The example in Listing~\ref{openshmem-hello-symmetric} shows a more complex \openshmem program that illustrates
-the use of symmetric data objects.  Note the declaration of the  \VAR{static
-short dest} array and its use as the remote destination in \openshmem short
-\PUT.  The use of the \VAR{static} keyword results in the \VAR{dest} array being
-symmetric on \ac{PE} \CONST{0} and \ac{PE} \CONST{1}.  Each \ac{PE} is able to
-transfer data to the \dest{} array by simply specifying the local address of the
-symmetric data object which is to receive the data.  This aids programmability,
-as the address of the \dest{} need not be exchanged with the active side
-(\ac{PE} \CONST{0}) prior to the RMA (Remote Memory Access) routine.
-Conversely, the declaration of the \VAR{short source} array is asymmetric.
-Because the \PUT{} handles the references to the \VAR{source} array only on the
-active (local) side, the asymmetric \source{} object is handled correctly.
+The example in Listing~\ref{openshmem-hello-symmetric} shows a more complex
+\openshmem program that illustrates the use of symmetric data objects.
+Note the declaration of the \VAR{static short dest} array and its use as the
+remote destination in \hyperref[subsec:shmem_put]{\FUNC{shmem\_put}}.
+
+The \VAR{static} keyword makes the \VAR{dest} array symmetric on all \acp{PE}.
+%% FIXME: The following sentence is not *technically* correct.
+Each \ac{PE} is able to transfer data to the \dest{} array by simply specifying
+the local address of the symmetric data object that will receive the data.
+This local address resolution aids programmability because the address of the
+\dest{} need not be exchanged with the active side (\ac{PE} \CONST{0}) prior to
+the RMA (Remote Memory Access) routine.
```

Comment:
`RMA (Remote Memory Access)` -> `\acf{RMA}` for "Remote Memory Access (RMA)" or just `\ac{RMA}`